src/main/resources/META-INF/plugin.xml

```diff
@@ -104,6 +104,14 @@
             key="inspection.not.null.checkable.property.source.used.with.default.property"
```

Comment:
I also would suggest to rename this to an explicit title key: 
e.g. `inspection.not.null.checkable.property.source.used.with.default.property.title=Constant or expression source property used with a default source property`

Currently the settings page displays the title like: `{0} property used with {1}`

Reply:
Changed